modules/graph_ql/main.tf

```diff
@@ -58,12 +187,22 @@ resource "aws_ecs_task_definition" "graphql_task" {
   container_definitions    = <<DEFINITION
 [
   {
-    "name": "orca-graphql",
+    "name": "orca-gql",
     "image": "ghcr.io/nasa/cumulus-orca/graphql:0.0.18",
     "cpu": 512,
     "memory": 256,
     "networkMode": "awsvpc",
+    "portMappings": [
+      {
+        "containerPort": ${local.graphql_port},
+        "hostPort": ${local.graphql_port}
+      }
+    ],
     "environment": [
+      {
+        "name": "PORT",
+        "value": "${local.graphql_port}"
+      }
```

Comment:
We would likely want to set the `ENV` value here to from a terraform variable that we would default to the Production instance and be able to override with a "Development" instance.

Reply:
Would it be acceptable to instead change the default `ENV` to `production`? Feels like that should really be our default, with overrides being used when additional leniency is desired for development.

Reply:
Changed default, per discussion.

Reply:
After thinking about it, wouldn't we still want to put the environment terraform variable in here so that you could change the behavior on deployment without any change to the underlying terraform? That way, as a developer you could deploy a development version while customers could deploy the production version without any code change for either just a terraform variable change?